test/functional/content_view/version/republish_repositories_test.rb

```diff
@@ -0,0 +1,30 @@
+require_relative '../../test_helper'
+require 'hammer_cli_katello/content_view_puppet_module'
+
+module HammerCLIKatello
+  describe ContentViewPuppetModule do
+    it 'allows republishing repositories on  a content view version' do
+      ex = api_expects(:organizations, :index) do |p|
```

Comment:
You can use the OrganizationHelpers https://github.com/Katello/hammer-cli-katello/blob/master/test/functional/organization/organization_helpers.rb